### ISSUE TITLE
NTR: 'diffuse alveolar damage'

### ISSUE THREAD

**Issue by paolaroncaglia** [obophenotype/human-phenotype-ontology#5626]
(Relevant for COVID19 studies)
Hi, as discussed with @matentzn , could you please add:

**Preferred term label:**
diffuse alveolar damage

**Synonyms**
Acronym (non-exact synonym): DAD

**Definition (free text, please give PubMed ID)**
An acute lung condition with the presence of hyaline membranes, made up of dead cells, surfactant, and proteins, and deposited along the walls of the alveoli, where gas exchange typically occurs, thereby making gas exchange difficult.
PMID:32364264 
PMID:28570160
Wikipedia:Diffuse_alveolar_damage

**Parent term (use hpo.jax.org/app)**
HP:0002088 Abnormal lung morphology
(not sure if a more specific parent exists)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Not sure

**Your nano-attribution (ORCID)**
0000-0003-0800-544X
0000-0002-2825-0621

Note, this is different from HP:0025420 'Diffuse alveolar hemorrhage'. In fact, the def of that term says "...Mild interstitial thickening, organizing pneumonia, or diffuse alveolar damage can also be seen.". Not sure if the two terms should be related in any way.

Thanks!

**Comment by matentzn** [comment]
@pnrobinson is it ok if we add this term?

**Comment by pnrobinson** [comment]
@mgriese, @katrinknoflach could you please comment?

**Comment by pnrobinson** [comment]
My initial feedback is that we do not observe "damage". The HPO tries to record the actual observed abnormalities. "damage" is our description of the consequence of these abnormalities. It would be better to describe these more precisely.

**Comment by mgriese** [comment]
Suggest 
Definition
DAD describes a histologic finding in the lung which includes dead cells, hyaline membrane made of surfactant and proteins filling the alveoli. This results in impaired gas exchange. The corresponding clinical entity is ARDS. DAD may result from pulmonary drug toxicity. 

I´ll comment later on diseases linked.

**Comment by paolaroncaglia** [comment]
Meanwhile, thank you both @pnrobinson and @mgriese for following up on this.

**Comment by mgriese** [comment]
Extending the definition a little bit and adding diseases associated:

Definition
DAD describes a comon histologic injury pattern of the lung. The early stages are characterized by epithelial cells necrosis and sloughing, fibrious exsudate, edema, and hyaline membranes made of surfactant and proteins, filling the alveoli. This results in impaired gas exchange. In later stages, type II cells and myofibroblasts proliferate within the interstitium and airspaces. The corresponding clinical entity is ARDS. DAD may result from pulmonary drug toxicity, occurs in immunosuppressed, severe viral infections, acute interstial pneumonitis and crack cocaine inhalation.

By the way, my ORCID number is: 0000-0003-0113-912X

**Comment by pnrobinson** [comment]
Adding as HP:0033006, thanks!
